cri-o-testonly.spec

```diff
@@ -0,0 +1,273 @@
+# This file is only used for running automated rpm build tests for PRs
+# The canonical dist-git source can be found at:
+# https://src.fedoraproject.org/rpms/cri-o/blob/master/f/cri-o.spec
+
+%if 0%{?with_debug}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package   %{nil}
+%endif
+
+%global provider        github
+%global provider_tld    com
+%global project         kubernetes-incubator
+%global repo            cri-o
+# https://github.com/kubernetes-incubator/cri-o
```

Comment:
Should this be up top or are you planning to add a global https field or some such here?

Reply:
actually it's not even needed, i just copied over the fedora spec file and deleted most of the stuff we didn't need to care about. I'll delete this line too